app/src/main/res/values/strings.xml

```diff
@@ -11,16 +11,29 @@
     <!-- Title for dialog displayed on first start -->
     <string name="welcome_title">Welcome to Syncthing for Android</string>
 
-    <!-- Text for dialog displayed on first start -->
+    <!-- Welcome wizard -->
+    <!-- Slide 1 -->
+    <string name="introduction">Introduction</string>
     <string name="welcome_text">Syncthing is an open-source file synchronization application.\n\
 To share data with other devices, you need to add their unique device IDs to the device list. Afterwards you can select which folders to share with which devices.\n\
 Please report any problems you encounter via Github.</string>
 
-    <string name="cont">Continue</string>
+    <!-- Slide 2 -->
+    <string name="storage_permission_title">Storage permission</string>
+    <string name="storage_permission_desc">Syncthing requires the storage permission to do file synchronization. We ask you to grant the permission.</string>
```

Comment:
As this is a required permission, I wouldn't ask and have a button - just explain and then request the permission on pressing "next".

Alternative wording that doesn't just repeat the title and seems "clearer" and easier to me:  
`Syncthing needs to access your storage to do file synchronization.`

Reply:
I think this summons a separate system dialog, hence has to be "initiated" somehow.

Agree with the better wording.